src/backend/distributed/shared_library_init.c

```diff
@@ -519,8 +519,8 @@ RegisterCitusConfigVariables(void)
 					 "be higher. This setting does not apply if the input format is "
 					 "binary."),
 		&SkipJsonbValidationInCopy,
-		true,
-		PGC_USERSET,
+		false,
```

Comment:
Hmm, why do we remove `PGC_USERSET`, it seems a separate value. I guess it should be the same as the above GUC?

```
	DefineCustomBoolVariable(
		"citus.enable_router_execution",
		gettext_noop("Enables router execution"),
		NULL,
		&EnableRouterExecution,
		true,
		**PGC_USERSET,
		GUC_NO_SHOW_ALL,**
		NULL, NULL, NULL);
```

Reply:
stupid mistake, fixed!